tests/bulk_create/models.py

```diff
@@ -6,6 +6,11 @@ class Country(models.Model):
     iso_two_letter = models.CharField(max_length=2)
 
 
+class Union(models.Model):
```

Comment:
This was the only thing I could think of that has a logical `ManyToMany` relation with `Countries`.

Come to think of it, another option might have been `Languages`, but that seems like a bit of a stretch.

If you have a better idea for a `ManyToMany` model for this test, please let me know.